packages/mandb.rb

```diff
@@ -17,17 +17,18 @@ class Mandb < Package
      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mandb-2.9.4-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: 'ccd36d83dc2dcb04d003a79fd503273ebd06a77d4da618f7033d537fda537d4e',
-     armv7l: 'ccd36d83dc2dcb04d003a79fd503273ebd06a77d4da618f7033d537fda537d4e',
-       i686: '97a79a235a9ab3c3f077ddf462a79d9d17ff3ff6cd35145321d790253775387f',
-     x86_64: '30939c206bd1adc66a33a8157d749b288a3726a995a8eb318c057104807c138d'
+    aarch64: '5248beb9649ac8d46050b64ae859380cd93dbdc38b3d66b89e1ea1bace6829f2',
+     armv7l: '5248beb9649ac8d46050b64ae859380cd93dbdc38b3d66b89e1ea1bace6829f2',
+       i686: '42694181223d691c6436b1a86912dd8578ccdf35c47bf3cff243e0daa1e7a7e1',
+     x86_64: '0f0265d2c924662d1f7099ddcc42876d516c29d18a985d2545e9b6dde7d81734'
```

Comment:
Let this be a lesson.  Don't ever rebuild binaries without bumping the version and especially with core packages. :)